routing/router_test.go

```diff
@@ -608,16 +611,18 @@ func TestSendPaymentErrorPathPruning(t *testing.T) {
 		t.Fatalf("unable to fetch source node pub: %v", err)
 	}
 
+	roasbeefLuoji := lnwire.NewShortChanIDFromInt(689530843)
```

Comment:
Where are these IDs coming from?

Reply:
From `basic_graph.json` 😛

Reply:
would recommend using `genIDs()` to ensure we don't reuse short chan ids

Reply:
This is different as the point of the test is to skip the hop between `roasbeef` and `luoji` and take an alternate path within the graph in `basic_graph.json`.

Reply:
Gotcha, I missed that this test also uses that graph. I had assumed you were reusing those chan ids in a diff test. I think should be okay then